### ISSUE TITLE
progressColor and controlColor prop not working 

### ISSUE THREAD

**Issue by coreybruyere** [FormidableLabs/spectacle#548]
```
<Slide transition={["zoom"]} bgColor="secondary" controlColor="red" align="flex-start" controlColor="red" progressColor="red">
  // Some stuff
</Slide>
```

I'm not able to change progress bar color and arrow color via `controlColor` and `progressColor` props provided w/ the Slide component. I've tried a few of the other props listed in the docs for the Slide component and don't have any issues with them, just those 2. Am I missing something?

**Comment by tizano** [comment]
Hello @coreybruyere, I don't know if you resolved the issue. But there is a workaround. You can set the progress bar's color by changing the quarternary theme's color.

For example :
```javascript
const theme = createTheme({
  primary: "#ffffff",
  secondary: "#1f2022",
  tertiary: "#03a9fc",
  quarternary: "#e1e1e1", // <-- change it
  bgOrange: "#f65b54",
  react: "#61dafb",
  bgReact: "#282c34"
});
```

and then set it to the Deck component
```javascript
<Deck transition={["zoom", "slide"]} transitionDuration={500} theme={theme}>
```

I hope it will be useful 😄

**Comment by ElreyB** [comment]
Hello @coreybruyere, I tried to recreate the issue using your code snippet but was able to change the colors of both arrow and progress bar. Are you still getting the same bug or has this been resolved?

```
<Slide
     transition={['zoom']}
     bgColor="secondary"
     align="flex-start"
     controlColor="red"
     progressColor="red"
       >
         Some stuff
       </Slide>
```
Before
![screen shot 2018-09-04 at 10 42 08 am](https://user-images.githubusercontent.com/20192033/45051395-d13ea180-b038-11e8-8c2f-5d42381e0079.png)

After

![screen shot 2018-09-04 at 10 42 57 am](https://user-images.githubusercontent.com/20192033/45051413-e0bdea80-b038-11e8-9503-4b029abbcb90.png)


 I will close this issue for now but if the issue persists write a comment and I will look more into it.  Side note, your have `controlColor=”red”` declare twice. This should not cause the issue but it worth removing just in case.

**Comment by ElreyB** [comment]
Closes #548
